Core/Parsing/ExpressionVisitors/TreeEvaluation/EvaluatableTreeFindingExpressionVisitor.cs

```diff
@@ -625,7 +630,7 @@ private bool IsCurrentExpressionEvaluatable (Expression expression)
     }
 #endif
 
-    private bool IsQueryableExpression (Expression expression)
+    private static bool IsQueryableExpression (Expression expression)
```

Comment:
As above, happy to revert. Interested to understand why you avoid statics like this.